packages/next/src/lib/mkcert.ts

```diff
@@ -53,10 +55,11 @@ async function downloadBinary() {
 
     Log.info(`Download response was successful, writing to disk`)
 
-    const arrayBuffer = await response.arrayBuffer()
-    const buffer = Buffer.from(arrayBuffer)
+    await pipeline(
+      Readable.fromWeb(response.body as any), // type definition of ReadableStream from @types/node and lib.dom.d.ts are incompatible
```

Comment:
Should we avoid `.fromWeb` as it's still marked as experimental stage 1? 

x-ref: https://nodejs.org/api/stream.html#streamreadablefromwebreadablestream-options

Reply:
Yes we should not use an experimental api.

All things considered, we should ditch the `fetch` here and instead use the significantly faster `undici.pipeline` or `undici.stream`. Keep it all in Node.js streams.

Reply:
If that is out of scope of this pr, I recommend manually reading the web stream returned by fetch and writing it to the file in chunks. 

You may also see if one of Node.js' write operations supports an async iterable and then you can probably pass the response body directly.

Reply:
@ijjk @Ethan-Arrowood Next.js has removed `undici` from its dependency, which means we no longer have access to `undici.pipeline` or `undici.stream`.